design-documents/graph-ql/coverage/b2b/company-credit.md

```diff
@@ -13,14 +13,15 @@ type Company {
 ###### Begin: Defining new types ######
 type CompanyCreditHistory {
     items: [CompanyCreditOperation]! @doc(description: "An array of company credit operations")
-    page_info: SearchResultPageInfo! @doc(description: "Metadata for pagination rendering")
-    total_count: Int @doc(description: "The number of the company credit operations matching the specified filter")
+    page_info: SearchResultPageInfo @doc(description: "Metadata for pagination rendering")
+    total_count: Int! @doc(description: "The number of the company credit operations matching the specified filter")
 }
 
 type CompanyCreditOperation {
+    uid: ID!  @doc(description: "Unique identifier") # id of the log entry
```

Comment:
we will have to add uid in all this schema as a lof the types don't have it still